config.json

```diff
@@ -3366,32 +3366,32 @@
     {
       "title": "Batteries Included",
       "content": "Epic & well-documented standard library. Need more? Use a PyPi package - there's one for everything.",
-      "icon": ""
+      "icon": "powerful"
```

Comment:
The icon for `powerful` might fit "Batteries Included" better than the icon for `productive`.